docs/tutorials/binning_slowly_changing_sources.ipynb

```diff
@@ -12,7 +12,8 @@
    "cell_type": "markdown",
```

Comment:
<p>Found another needed change; we're no longer using the same set of lightcurves from the previous notebook, so the second sentence should be removed or replaced accordingly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Ah good catch! Opted to remove.